ebpf/c/tracer-ebpf.c

```diff
@@ -406,15 +432,13 @@ static int read_conn_tuple(conn_tuple_t* tuple, tracer_status_t* status, struct
     // Check if we can map IPv6 to IPv4
     if (family == CONN_V6 && is_ipv4_mapped_ipv6(saddr_h, saddr_l, daddr_h, daddr_l)) {
         tuple->metadata |= CONN_V4;
+
```

Comment:
should we also reset `saddr_h` and `laddr_h` here ? Just to avoid errors in the future